src/event.c

```diff
@@ -154,7 +156,7 @@ void xmpp_run_once(xmpp_ctx_t *ctx, unsigned long timeout)
             conn->error = ECONNABORTED;
             conn_disconnect(conn);
         }
-
+    NEXT_ITEM:
```

Comment:
Please, don't indent label (remove spaces at the beginning).

Reply:
It was auto-indented. Fixed the clang-format file.